### ISSUE TITLE
Visual Studio Code - Terminal Blank Screen

### ISSUE THREAD

**Issue by anesta95** [microsoft/vscode#81295]
- VSCode Version: 1.38
- OS Version: Windows 10 Version 1903 OS Build 18362.356
Hello all,

Have seen this bug pop up on a few other issue reports but have not come across a remedy for it yet. Basically, as of this weekend, I am no longer able to run Python scripts in VS Code's integrated python terminal. The terminal will show up blank when the script is run and auto-generate a cmd window (preference I set) which does not execute the script either. However, I can run the code in the Python Interactive Window. Additionally, even my cmd integrated terminal in VS Code is showing up as 'blank' which is making me think it might be a bug of some sort on one of the new updates. I am using a Python interpreter downloaded locally on my computer (idk if that really matters)
![image](https://user-images.githubusercontent.com/44952222/65394762-6d7c3900-dd5f-11e9-980a-14f495176558.png)
![image](https://user-images.githubusercontent.com/44952222/65394774-9270ac00-dd5f-11e9-8e50-3617b6d9b1c7.png)
Here are the specifics of my settings.json file right now:
![image](https://user-images.githubusercontent.com/44952222/65394790-bcc26980-dd5f-11e9-976f-b0158bcd1f9c.png)
Any help would be supremely appreciated. I'm kind of at my wit's end and would like to keep my hair attached to my scalp for the time being. Thank you in advance :-)
Steps to Reproduce:

1. Uninstall/reinstall VS Code and only used the Python extension
2. Tried to find settings.json command related to integrated terminal

Does this issue occur when all extensions are disabled?: Yes/No
Python Extension needed to run code in the integrated terminal. Uninstall/reinstall with no other extensions besides the Python extension yields the same result.

**Comment by kieferrm** [comment]
@Tyriar pls close if caused by the python extension.

**Comment by Tyriar** [comment]
Can you open devtools and check the console for errors after opening a terminal like this?

Also can you enable escape sequence logging and try opening and paste in the results, if any? https://github.com/microsoft/vscode/wiki/Terminal-Issues#enabling-escape-sequence-logging

**Comment by anesta95** [comment]
Hey @Tyriar sorry responding so late. So I opened the DevTools console and everything looks to be working okay. However I am not super familiar with that environment so I very well could be missing something obvious. I also enabled sequence logging and checked if there were any results both in the integrated vscode terminal or the cmd that autogenerates when I open vs code and there were none.

![image](https://user-images.githubusercontent.com/44952222/65904002-5eeedc80-e372-11e9-974a-e5eac4637716.png)

![image](https://user-images.githubusercontent.com/44952222/65904051-7a59e780-e372-11e9-8ff9-48665ab2a8e4.png)

![image](https://user-images.githubusercontent.com/44952222/65904067-8645a980-e372-11e9-95c3-1f3d4d1394a6.png)

![image](https://user-images.githubusercontent.com/44952222/65904140-ae350d00-e372-11e9-9058-b64456d0d1c2.png)

Let me know how I should proceed and thank you for the help :-)
